### ISSUE TITLE
How to install this?

### ISSUE THREAD

**Issue by imim332** [geofs-plugins/voice-atc#1]

**Comment by soccernicolas** [comment]
@ImranDoet 
1. Install TamperMonkey from the chrome web store
2. click clone or download on the Code tab here
3. Click Download ZIP
4. Extract the folder in the zip
5. Open TamperMonkey and create a new script
6. Remove everything, then drag and drop geofs_plugin.js in
7. That's all! There's one quick thing I'll do a pull request on, but your good after that.

**Comment by TBA1608** [comment]
it says that it is an invalid script 
IDK what to do cuz ctrl+ q is orbit do I change the keybind?

**Comment by TBA1608** [comment]
@soccernicolas

**Comment by TBA1608** [comment]
doesn't matter its prob not supported

**Comment by sushrey** [comment]
yes it says as its an invalid script. but then i did not erase everything in new script and instead only copied and pasted the usercript part and filled out the other above details manually. then it accepted the userscript but then i see no differrence yet in geofs. when i press ctrl + q the camera starts revolving around the plane thats all

**Comment by latam5** [comment]
for me when i enter the frequency, it says404 page
Is that normal?
Latam

**Comment by W72702** [comment]
@latam5 this plugin no longer works

**Comment by the-V01D** [comment]
Im guessing it just was not maintained

**Comment by W72702** [comment]
Unfortunately yes. I have made a pull request with a working website so we will see if that changes
